pkg/cmd/server/start/master_args.go

```diff
@@ -51,6 +52,9 @@ type MasterArgs struct {
 	StartControllers bool
 	PauseControllers bool
 
+	// DNSDomain is the authoritative domain of the DNS server.
```

Comment:
The same option is available in the node config, but not in the node launch arguments